crypto/helpers.go

```diff
@@ -22,11 +22,15 @@ import (
 	"strings"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/shopspring/decimal"
 )
 
 // Myst represents a single myst ERC 777 token.
 const Myst uint64 = 1000_000_000_000_000_000
 
+// MystSize defines MYST size - precision for floating points.
+const MystSize uint = 18
```

Comment:
if you define it as uint but use it as int32, maybe define a `mystSizeInt32` just below and use that wherever you're using it now?

Reply:
sure 🤷🏻